src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let result = days * 40;
+
+  if (days >= 7) {
+    result -= 50;
+  } else if (days >= 3) {
+    result -= 20;
+  }
```

Comment:
It's a code style issue. You can simplify your conditionals by removing the `else` keyword and using multiple `if` statements with `return`. This makes your function easier to read and understand. Also, it reduces the amount of indentation in your code.